lib/emerald/nodes/list.rb

```diff
@@ -4,22 +4,15 @@
 require 'treetop'
 require_relative 'node'
 
-# TODO: Refactor this and update doc comment
-# Multiple head elements I think
-class List < Node
+# Special rule for lists of images, styles, and scripts
+class ListSpecial < Node
   def to_html(_context)
-    if elements[0].text_value == 'styles'
-      elements[4].elements.map do |e|
-        "<link rel='stylesheet' href=#{e.text_value.strip} />"
-      end.join("\n")
-    elsif elements[0].text_value == 'scripts'
-      elements[4].elements.map do |e|
-        "<script type='text/javascript' src=#{e.text_value.strip}></script>"
-      end.join("\n")
-    elsif elements[0].text_value == 'metas'
-      elements[4].elements.map do
-        ''
-      end.join("\n")
-    end
+    elements[4].elements.map do |e|
+      case elements[0].text_value
```

Comment:
We should add labels to these in the grammar so we aren't referring to them by index